cli/installer/install-azd.ps1

```diff
@@ -341,6 +345,16 @@ try {
 
     try {
         if (isLinuxOrMac) {
+            if ($IsMacOS -and (-not $SkipVerify)) {
+                Write-Verbose "Verifying signature of $binFilename" -Verbose:$Verbose
+                codesign -v "$tempFolder/decompress/$binFilename" *>$null
+                if (-not $?) {
+                    Write-Error "Could not verify signature of $binFilename"
```

Comment:
You should actually capture the output and say why. E.g.,

```pwsh
$result = codesign -v "$tempFolder/decompress/$binFilename" 2>&1
if (!$?) {
  Write-Error "Could not verify signature of $binFilename: $result"
  # ...
}

Reply:
That could provide useful information.

Reply:
That's a good callout. Will address.